### ISSUE TITLE
What is the validation accuracy rate?

### ISSUE THREAD

**Issue by sugiantonehemia** [nikhil-salodkar/facial_expression#1]
Hi Nikhil,
I am interested in the project you have done since I am doing my research on AffectNet dataset. I am wondering what is the validation accuracy rate after you trained using the dataset? I tried to train my fine-tuned ResNet50 model but it stuck to 70% validation accuracy rate.

**Comment by nikhil-salodkar** [comment]
70% validation accuracy rate on full validation set as provided by the creators seems to be quite good. It is much better than the baseline validation accuracy that the creators got as mentioned in their paper.
I did not fine tune the model but just used pretrained ResNet50 model as provided in Keras, kept first half of the weights fixed and trained last half of the layers in the model. By doing this the top 1 validation accuracy turned out to be only 57% and top 2 accuracy as 78%. I also only used 80% of the manually annotated training set and kept rest as test set.
If you read their paper, it is mentioned that this dataset is complex and even two human annotators agreement rate is not more than approximately 75%.
